test/helper.rb

```diff
@@ -772,6 +772,15 @@ def mock_connection
     @connection
   end
 
+  def random_index_hash seed = nil, index_id = nil
```

Comment:
I think the args should be just `index_id`. If index_id is nil then it should generate a random id.

```ruby
def random_index_hash index_id = nil
  index_id ||= "example-index-#{rand 9999}"
  {
    "projectId" => project,
    "indexId" => index_id
  }
end
```